docs/using-redis.md

```diff
@@ -0,0 +1,93 @@
+# Gluster Volume as persistent store for running Redis server
+
+## Create PersistentVolumeClaim for redis server
+
+Create the yaml file(`volume-redis-app.yaml`) as below by specifying
+the claim size as required
+
+```
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: redis-server-vol
+  annotations:
+    volume.beta.kubernetes.io/storage-class: glusterfs-csi
```

Comment:
Please change this is the modern syntax for specifying the StorageClass. See: https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/#using-dynamic-provisioning